kpconv_torch/io/ply.py

```diff
@@ -190,7 +190,7 @@ def read_ply(filepath, triangular_mesh=False):
         ):
```

Comment:
Be careful of remaining `properties[4][0]` and `properties[6][0]` (Github prevents me to propose a suggestion as the diff is not on the corresponding line).